### ISSUE TITLE
generate BOM csv for documentation

### ISSUE THREAD

**Issue by wassfila** [Informaticore/SuperPower#31]
the generated documentation can include CSV files as they are, so can take advantage to keep documenting the latest version.

**Comment by Informaticore** [comment]
for that generation we might clean up the components first in KiCad. You remember that table I was talking about once? @wassfila

**Comment by wassfila** [comment]
> 
> 
> for that generation we might clean up the components first in KiCad. You remember that table I was talking about once? @wassfila

right, for the component folder I have #28 I'll comment about it there
